bedrock/firefox/templates/firefox/election/index.html

```diff
@@ -16,152 +16,298 @@
+                      data-link-type="download"
+                      data-download-os="Desktop"
+                      data-download-version="firefox election bundle win"
+                      data-download-location="election page">
+                      Download Firefox Election Bundle
+                    </a>
+                  </div>
+                  <div id="case-funnel-mac" class="hidden">
+                    <a
+                      class="download-link button mzp-c-button mzp-t-download"
+                      href="https://download.mozilla.org/?product=firefox-election-edition&os=osx&lang=en-US"
+                      data-link-type="download"
+                      data-download-os="Desktop"
+                      data-download-version="firefox election bundle mac"
+                      data-download-location="election page">
+                      Download Firefox Election Bundle
+                    </a>
+                  </div>
+                  <p><a href="{{ url('firefox.new') }}" data-button-name="just firefox">Nah, just give me Firefox</a></p>
+                  <small class="mzp-c-button-download-privacy-link">
+                    <a href="{{ url('privacy.notices.firefox') }}" >Firefox Privacy Notice</a>
+                  </small>
+                </div>
+              </div>
             </div>
           </div>
         </div>
       </section>
-      <section class="mzp-c-call-out mzp-t-product-firefox mzp-t-firefox t-container">
+    </div>
+    <div id="case-firefox-desktop-current">
+      <div class="t-paper">
+        <section class="mzp-c-hero">
+          <div class="mzp-l-content">
+            <div class="mzp-c-hero-body">
+              <h1 class="mzp-c-hero-title">Get Smart Before You Vote.</h1>
+              <div class="mzp-c-hero-desc">
+                  <p>Decode political ad targeting and keep Facebook off your trail with these extensions.</p>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section class="mzp-c-call-out mzp-t-product-firefox mzp-t-firefox t-container">
+          <div class="mzp-l-content">
+            <h2 class="mzp-c-call-out-title">The Firefox Facebook Container</h2>
+            <div class="mzp-c-call-out-desc">
+              <p>Facebook helps political advertisers target you by tracking your activity on and off their platform.</p>
+              <p>This extension makes it easier to explore the web without Facebook watching, so you can keep your political opinions to yourself and your dinner companions. </p>
+              <p class="mzp-c-button-download">
+                <a class="mzp-c-button" rel="external" href="https://addons.mozilla.org/en-US/firefox/addon/facebook-container/" data-button-name="Facebook Container Get It">Get It</a><br>
+                <small class="mzp-c-button-download-privacy-link">
+                  <a href="{{ url('privacy.notices.firefox') }}">Firefox Privacy Notice</a>
```

Comment:
Oh I see the link is here as well... so I'm guessing since this addon really just uses a built in Firefox feature it's covered by the same privacy policy and legal requires the link paired with the addon? In which case, as you were 😄